From b8e5a8cff4300dd14bf7685ceb114df0fd775952 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 10:12:49 +0100
Subject: [PATCH 26/53] apps: nagle: remove unused af_family variable

The pf_class variable is being used instead, so just remove
the other one.

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/apps/nagle_rcv.c | 5 +----
 src/apps/nagle_snd.c | 7 +------
 2 files changed, 2 insertions(+), 10 deletions(-)

diff --git a/src/apps/nagle_rcv.c b/src/apps/nagle_rcv.c
index 47695e4..cb2ea8f 100644
--- a/src/apps/nagle_rcv.c
+++ b/src/apps/nagle_rcv.c
@@ -81,8 +81,7 @@ main(int argc, char *argv[])
         struct iovec iov;
         struct msghdr inmessage;
 	char incmsg[CMSG_SPACE(sizeof(sctp_cmsg_data_t))];
-        int error;
-	int pf_class, af_family;
+	int error, pf_class;
 	char *big_buffer;
 	char *local_host = NULL;
 	int local_port = SCTP_TESTPORT_1; 
@@ -161,7 +160,6 @@ main(int argc, char *argv[])
                 exit(1);
         }
 	pf_class = PF_INET6;
-	af_family = AF_INET6;
 
         host.v6.sin6_family = AF_INET6;
         memcpy(&host.v6.sin_addr, hst->h_addr_list[0], hst->h_length);
@@ -174,7 +172,6 @@ main(int argc, char *argv[])
                 exit(1);
         }
 	pf_class = PF_INET;
-	af_family = AF_INET;
 
         host.v4.sin_family = AF_INET;
         memcpy(&host.v4.sin_addr, hst->h_addr_list[0], hst->h_length);
diff --git a/src/apps/nagle_snd.c b/src/apps/nagle_snd.c
index 1340fa7..825eb28 100644
--- a/src/apps/nagle_snd.c
+++ b/src/apps/nagle_snd.c
@@ -93,8 +93,7 @@ main(int argc, char *argv[])
 	struct sctp_sndrcvinfo *sinfo;
         struct iovec out_iov;
         char *message;
-        int error, bytes_sent;
-	int pf_class, af_family;
+	int error, bytes_sent, pf_class;
         sctp_assoc_t associd;
 	uint32_t ppid;
 	uint32_t stream;
@@ -218,8 +217,6 @@ main(int argc, char *argv[])
         target.v6.sin6_port = htons(remote_port);
 
         pf_class = PF_INET6;
-        af_family = AF_INET6;
-
 #else
         hst = gethostbyname(local_host);
         if (hst == NULL || hst->h_length < 1) {
@@ -241,8 +238,6 @@ main(int argc, char *argv[])
         target.v4.sin_port = htons(remote_port);
 
         pf_class = PF_INET;
-        af_family = AF_INET;
-
 #endif /* TEST_V6 */
 
         sk = test_socket(pf_class, SOCK_SEQPACKET, IPPROTO_SCTP);
-- 
1.7.11.7

